workitem/simple_type.go

```diff
@@ -38,12 +38,12 @@ func (t SimpleType) GetKind() Kind {
 var timeType = reflect.TypeOf((*time.Time)(nil)).Elem()
 
 // ConvertToModel implements the FieldType interface
-func (fieldType SimpleType) ConvertToModel(value interface{}) (interface{}, error) {
+func (t SimpleType) ConvertToModel(value interface{}) (interface{}, error) {
```

Comment:
ok, I can revert that. I just intended to clear a compilation warning here, but I agree it has nothing to do with this issue.